src/test/java/Response/ResponseTest.java

```diff
@@ -0,0 +1,59 @@
+import static org.junit.Assert.assertEquals;
+import org.junit.Before;
+import org.junit.Test;
+
+public class ResponseTest {
+
+  private Response goodResponse = new Response();
+  private Response badResponse = new Response();
```

Comment:
This doesn't seem to be used anywhere in these tests.  I would also rename `goodResponse` as this inaccurately implies that Response performs some validation and determines what's "good" or "bad"

Reply:
I've changed the name to `response`.